README.md

```diff
@@ -1,61 +1,107 @@
-# WPC: WordPress & (Docker) Compose
+# WPC: WordPress in Containers
 
-## Installation
+This tool is used to add shell scripts and configuration files that allow running WordPress in Docker containers.
 
-The easiest way to "install" wpc is to create an alias in your shell:
+If the repository you're using already has `wpc` you'll find the following files:
+
+- `script/server` - run WordPress in a container
+  - WordPress will be running on http://localhost
+  - MailCatcher will be running on http://localhost:1080
+  - Beanstalk Console will be running on http://localhost:2080
+- `script/setup` - install WordPress
+  - the default user is `admin` with password `admin`
+  - will also install [whippet](https://github.com/dxw/whippet) dependencies
+  - setup configuration files are in the `setup/` directory
+- `script/update` - run any updates after checking out the latest version
+  - this will update whippet dependencies to the latest versions
+- `script/console` - open a `/bin/sh` console inside the main continer
+- `script/bootstrap` - update dependencies (this is called by `script/setup` and `script/update`)
+- `bin/wp` - run [wp-cli](https://wp-cli.org/)
+- `docker-compose.yml` - the Docker Compose configuration
+- `config/server.php` - configuration that would normally go in `wp-config.php` (checked into the repository)
+- `config/server-local.php` - configuration that would normally go in `wp-config.php` (not checked into the repository)
+- `setup/internal.sh` - this is run by `script/setup`, it runs the wp-cli commands to install WordPress
+
+## Requirements
+
+- [Docker](https://docs.docker.com/docker-for-mac/install/)
+- [Docker Compose](https://docs.docker.com/compose/install/)
```

Comment:
The Docker Desktop installation now includes Docker Compose. So this wouldn't be needed if they'd installed Docker via the previous link.